### DISCUSSION TITLE
No provider available for the service

### DISCUSSION THREAD

**Discussion by Hwangsome** [apache/dubbo#13877] [Question]
dubbo 版本： 3.1.6
机器： windows 11
服务端启动并完成注册到nacos.
![image](https://github.com/apache/dubbo/assets/56013761/78d13692-b81b-4e02-8cd9-3ab2feed0592)
jdk version: 1.8
协议： GRPC

问题复现：
使用dubbo 3.1.6, grpc 1.54.1,protobuf version 3.21.7,启动服务端成功之后，启动consumer报错：
`Failed to check the status of the service com.bh.dubbo.model.DubboDemoServiceGrpc___IDemoService. No provider available for the service com.bh.dubbo.model.DubboDemoServiceGrpc___IDemoService from the url consumer://192.168.2.121/com.bh.dubbo.model.DubboDemoServiceGrpc___IDemoService?application=dubbo-admin&background=false&dubbo=2.0.2&generic=true&interface=com.bh.dubbo.model.DubboDemoServiceGrpc___IDemoService&logger=slf4j&pid=26004&qos.port=32222&register.ip=192.168.2.121&release=3.1.6&side=consumer&sticky=false&timestamp=1691290215726&unloadClusterRelated=false to the consumer 192.168.2.121 use dubbo version 3.1.6`

**Comment by Hwangsome** [comment]
dubbo 3.2.0启动服务端和消费端均不报错，但是消费端去调用服务端的时候报错：
```
org.apache.dubbo.rpc.StatusRpcException: UNIMPLEMENTED : Method not found: com.bh.dubbo.model.DubboDemoServiceGrpc$IDemoService/demo

	at org.apache.dubbo.rpc.TriRpcStatus.asException(TriRpcStatus.java:214)
	at org.apache.dubbo.rpc.protocol.tri.call.UnaryClientCallListener.onClose(UnaryClientCallListener.java:51)
	at org.apache.dubbo.rpc.protocol.tri.call.TripleClientCall.onComplete(TripleClientCall.java:112)
```

**Comment by Hwangsome** [comment]
![image](https://github.com/apache/dubbo/assets/56013761/ae80a67d-2b46-42ca-b97e-da07d62259e4)
这是dubbo-admin中的，可以看到服务也注册进去了。

**Comment by xiaoxuxua** [comment]
请问您解决了么，我也遇到了这个问题
